alias/dlgalias.c

```diff
@@ -174,6 +176,45 @@ static int alias_data_observer(struct NotifyCallback *nc)
   return 0;
 }
 
+/**
+ * alias_sort_observer - Listen for `sort_alias` configuration changes and reorders alias list
+ */
+static int alias_sort_observer(struct NotifyCallback *nc)
+{
+  if (!nc->event_data)
+    return -1;
+  if (nc->event_type != NT_CONFIG)
+    return 0;
+
+  struct EventConfig *ec = nc->event_data;
+
+  if (!mutt_str_equal(ec->name, "sort_alias"))
+    return 0;
+
+  struct AliasMenuData *mdata = nc->global_data;
+
+  menu_data_sort(mdata);
+
+  return 0;
+}
+
+/**
+ * alias_color_observer - Listen for color configuration changes and refreshes the menu
+ */
+static int alias_color_observer(struct NotifyCallback *nc)
+{
+  if ((nc->event_type != NT_COLOR) || !nc->event_data || !nc->global_data)
+    return -1;
+
+  if (nc->event_subtype != MT_COLOR_STATUS)
+    return 0;
```

Comment:
Although the Address Book doesn't have any custom colours, there are three that affect us:
- `MT_COLOR_INDICATOR`
- `MT_COLOR_NORMAL`
- `MT_COLOR_STATUS`

We _could_ check for each of them, but I think we should just drop the `if`.
If there are multiple `NT_COLOR` events, all we're doing is setting a flag.

Reply:
Fixed!